app/src/main/java/com/example/superheroes/HeroesScreen.kt

```diff
@@ -0,0 +1,140 @@
+package com.example.superheroes
+
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.navigationBarsPadding
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.statusBarsPadding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material3.Card
+import androidx.compose.material3.CardDefaults
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.example.superheroes.data.model.Hero
+import com.example.superheroes.data.repository.HeroesRepository
+import com.example.superheroes.ui.theme.Shapes
+import com.example.superheroes.ui.theme.SuperheroesTheme
+
+@Composable
+fun HeroesScreen() {
+    HeroesList(heroes = HeroesRepository.heroes + HeroesRepository.heroes)
```

Comment:
`HeroesRepository.heroes + HeroesRepository.heroes` is here just to be able to scroll the list ;)